src/main/java/com/commercetools/sync/commons/utils/ChunkUtils.java

```diff
@@ -0,0 +1,77 @@
+package com.commercetools.sync.commons.utils;
+
+import static com.commercetools.sync.commons.utils.CompletableFutureUtils.collectionOfFuturesToFutureOfCollection;
+import static java.util.stream.Collectors.toList;
+
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereRequest;
+import io.sphere.sdk.queries.PagedQueryResult;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+import javax.annotation.Nonnull;
+
+public class ChunkUtils {
```

Comment:
Add unit tests to cover this.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/667/commits/3a0e7a7bab6c96342667a7910bb9d121bfba2206